github/handlers/ExecuteBlockActionHandler.ts

```diff
@@ -1010,40 +1014,83 @@ export class ExecuteBlockActionHandler {
                             ).roomId;
                             room = await this.read.getRoomReader().getById(roomId) as IRoom;
                         }
-                        let githubissueStorage = new GithubRepoIssuesStorage(this.persistence,this.read.getPersistenceReader());
-                        let repoIssuesData: IGitHubIssueData = await githubissueStorage.getIssueData(room?.id as string,user);
-                            if(repoIssuesData?.issue_list?.length){
-                                let index = -1;
-                                let currentIndex = 0;
-                                for(let issue of repoIssuesData.issue_list){
-                                    if(issue.issue_id == issueId ){
-                                        index=currentIndex;
-                                        break;
-                                    }
-                                    currentIndex++;
-                                }
-                                if(index !== -1){
-                                    repoIssuesData.issue_list[index].share=false;
-                                    await githubissueStorage.updateIssueData(room as IRoom,user,repoIssuesData);
+                        let githubissueStorage = new GithubRepoIssuesStorage(this.persistence, this.read.getPersistenceReader());
+                        let repoIssuesData: IGitHubIssueData = await githubissueStorage.getIssueData(room?.id as string, user);
+                        if (repoIssuesData?.issue_list?.length) {
+                            let index = -1;
+                            let currentIndex = 0;
+                            for (let issue of repoIssuesData.issue_list) {
+                                if (issue.issue_id == issueId) {
+                                    index = currentIndex;
+                                    break;
                                 }
-                                 let data = {
-                                    issues: repoIssuesData.issue_list,
-                                    pushRights : repoIssuesData.push_rights, //no access token, so user has no pushRights to the repo,
```

Comment:
@VipinDevelops  All these changes look unrelated to this PR. Can you please verify

Reply:
Yeah I removed all the formatting changes please have a look :smile: